onnxscript/test/converter_test.py

```diff
@@ -539,7 +546,7 @@ def check_run(self, onnxfn, inputs, expected_output):
         np.testing.assert_equal(output, expected_output)
 
         # Test running model in eager mode
-        output = onnxfn(*inputs)
+        output = onnxfn._usercall(*inputs)
```

Comment:
Wondering if there will be a public method for this?

Reply:
I am not sure. There is a technical question: there are two conventions for argument/result types. Whether we can rationalize this and automate this or not is to be seen. I would like to hide it if possible, but am not sure we will always be able to.

Reply:
Ok, eliminated/rewrote it.

Reply:
Awesome thanks:) just asking questions for my education as well. I was wondering what _usercall does. Is the new version doing the same thing?

Reply:
So, the main difference is: eager-mode execution of onnxscript functions use our own Tensor type, which is a simple wrapper around numpy arrays, used, for example, to define our own overloaded ops. However, at the top-level (user-facing), it is convenient to remove the wrapper and return numpy arrays. That's main difference between `_usercall` and `_libcall`. (I think there is some cleaning up to do with regards to these methods, which I will do as a separate PR, as it is unrelated to current PR.)

Reply:
Thanks for explaining!

Reply:
Any other concerns? The PR still needs an approval to be merged :)